project/src/main/java/com/google/sps/Models/User.java

```diff
@@ -31,8 +31,4 @@ public String getEmail() {
   public ArrayList<String> getDocs() {
       return docHashes;
   }
-
-  public void setDocs(ArrayList<String> newHashes) {
```

Comment:
Is a user's hashes never being updated?

Reply:
the user's hashes should be updated in the datastore; there should be no set methods in the User class since it wouldn't persist when you retrieve it again

Reply:
Okay so what I am understanding is that since we are creating new instances of the user class each time that we reload a page, this shouldn't be an issue since it will be updated in the new instance?

Reply:
correct; every pull from the database should be correct.